intelmq/lib/bot.py

```diff
@@ -256,18 +258,19 @@ def __disconnect_pipelines(self):
             self.__destination_pipeline = None
             self.logger.info("Disconnecting from destination pipeline.")
 
-    def send_message(self, message):
-        if not message:
-            self.logger.warning("Sending Message: Empty message found.")
-            return False
+    def send_message(self, *messages):
+        for message in messages:
+            if not message:
```

Comment:
I didn't change this part, but I do think it's useful: It's a situation which SHOULD NOT happen, but can be handled -> warning.